crates/containerd-shim-wasm/src/sandbox/shim/otel.rs

```diff
@@ -136,11 +136,16 @@ impl Config {
             Protocol::Grpc => self.init_tracer_grpc(),
         };
 
-        opentelemetry_otlp::new_pipeline()
+        let tracer = opentelemetry_otlp::new_pipeline()
             .tracing()
             .with_exporter(exporter)
-            .with_trace_config(sdktrace::config())
-            .install_batch(runtime::Tokio)
+            .with_trace_config(Default::default())
+            .install_batch(runtime::Tokio)?
+            .tracer_builder("opentelemetry-otlp")
```

Comment:
I wonder if we want to use this name (which follows the previous behaviour of `install_batch`), or something runwasi specific. @Mossaka ?

Reply:
where is this name shown up in the jeager UI? I think it makes more sense for it to be runwasi specific.

Reply:
went with `containerd-wasm-shim`